src/3box.js

```diff
@@ -145,6 +145,29 @@ class Box {
         setTimeout(() => { spaceMessageFilterActive = false }, 3000)
       }
       if (data.type === 'HAS_ENTRIES') {
+        console.log(data)
```

Comment:
This line should be removed before this PR is merged.

Reply:
other logs below as well